components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -252,16 +223,29 @@ public void printHeader() {
         System.out.println(String.join(",", labels));
     }
 
+    private void printBuildValidationData(List<BuildValidationData> buildValidationData) {
```

Comment:
Maybe use 
- `printBuildValidationDataItems` and `printBuildValidationDataItem` or
- `printAllBuildValidationData` and `printSingleBuildValidationData`

Reply:
✅ Done